projects/components/src/utils/test/datagrid/filter-utils.ts

```diff
@@ -75,9 +75,9 @@ export function createDatagridFilterTestHelper<V, C>(
     const grid = finder.find(ClrDatagridWidgetObject);
 
     wf.hostComponent.setFilter(filterType, wf, config || ({} as C));
-    grid.unwrap().fixture.detectChanges();
+    grid.component.fixture.detectChanges();
```

Comment:
I think we still want to rework the filter tests to not rely on using component instances and to be built into the datagrid, but it might be a little much for this review. Can you add a ticket to our backlog titled something like "Rework datagrid filter tests to use WidgetObject rather than componentInstance"?